src/components/BottomBar/Origin/OriginPicker.jsx

```diff
@@ -2,98 +2,225 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
 import SmallLabel from '../../common/SmallLabel/SmallLabel';
-import Icon from '../../common/Icon/Icon';
 import LoadingString from '../../common/LoadingString/LoadingString';
 import Picker from '../Picker';
 import Popover from '../../common/Popover/Popover';
+import Button from '../../common/Input/Button/Button';
 import FilterList from '../../common/FilterList/FilterList';
-import DataManager from '../../../api/DataManager';
-import { NavigationAnchorKey, NavigationAimKey, RetargetAnchorKey, RetargetAimKey } from '../../../api/keys';
+import {
+  NavigationAnchorKey,
+  NavigationAimKey,
+  RetargetAnchorKey,
+  RetargetAimKey,
+  ScenePrefixKey
+} from '../../../api/keys';
 import FocusEntry from './FocusEntry';
 
-import Earth from './images/earth.png';
+import {
+  setNavigationAction, 
+  setOriginPickerShowFavorites,
+  setPopoverVisibility
+} from '../../../api/Actions';
+
 import styles from './OriginPicker.scss';
 
-const icons = {
-  Earth,
-};
```

Comment:
storyTouch version might need this?

Reply:
This code is not used in storyTouch - this is the regular webgui component